### ISSUE TITLE
Import @grammyjs/types downloads old version

### ISSUE THREAD

**Issue by AlexOwl** [grammyjs/menu#30]
https://esm.sh/@grammyjs/types@v2 redirects to 2.0.0
https://esm.sh/@grammyjs/types@2 redirects to newest 2.12.1

I think it was mistakenly typed v2, because it's necessary to get new patches (including security one)

```
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: macos aarch64
Version: 1.30.2
Args: ["/opt/homebrew/bin/deno", "lsp"]

thread 'main' panicked at 'internal error: entered unreachable code', cli/lsp/documents.rs:1455:7
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[Error - 9:07:58 PM] Client Deno Language Server: connection to server is erroring. Shutting down server.
[Error - 9:07:58 PM] Connection to server got closed. Server will not be restarted.
```

Pull request #29

**Comment by dcdunkan** [comment]
I understand that the version needs to be updated. But FYI, deno panicking is similar to https://github.com/denoland/deno/issues/17640 and its fix have been released as Deno 1.30.3

**Comment by KnorpelSenf** [comment]
@AlexOwl does this still happen with 1.30.3?

**Comment by KnorpelSenf** [comment]
Please also review #31.

**Comment by KnorpelSenf** [comment]
Closed by #29.
